app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/ScheduleData.kt

```diff
@@ -1,24 +1,50 @@
 package nerd.tuxmobil.fahrplan.congress.models
 
+/**
+ * Represents schedule data of one or multiple rooms
+ * for a single day specified by its [dayIndex].
+ */
 data class ScheduleData(
         val dayIndex: Int,
         val roomDataList: List<RoomData>
 ) {
+
+    companion object {
+        const val UNKNOWN_ROOM_INDEX = -1
+    }
+
+    /**
+     * Returns the total number of rooms.
+     */
     val roomCount: Int
         get() = roomDataList.size
 
+    /**
+     * Returns an unordered list of room names.
```

Comment:
`roomDataList` is in display order. Then so is the list returned from this.

Reply:
Side note: A `List` is always ordered. It might not be sorted, though.

Reply:
I would like to make this very clear for the user of this class. Therefore, I would document at the constructor argument that one needs to pass `roomDataList` in display order since the class does not take care of it. Do you agree this is a good place?

Reply:
Updated. See 952d3d71cb2c9a1dcc921a712ddb1ab64a1eab20.